monitors/SyntheticPegMonitor.js

```diff
@@ -86,6 +102,102 @@ class SyntheticPegMonitor {
     }
   };
 
+  // Checks difference between minimum and maximum historical price over `volatilityWindow` amount of time.
+  // Fires a message if the difference exceeds the `volatilityAlertThreshold` %.
+  checkPegVolatility = async () => {
+    const pricefeed = this.medianizerPriceFeed;
+
+    this.logger.debug({
+      at: "SyntheticPegMonitor",
+      message: "Checking peg price volatility"
+    });
+
+    const { pricefeedVolatility, pricefeedLatestPrice } = await this._checkPricefeedVolatility(pricefeed);
+    if (!pricefeedVolatility || !pricefeedLatestPrice) {
+      this.logger.warn({
+        at: "SyntheticPegMonitor",
+        message: "Unable to get price",
+        pricefeed: "Medianizer",
+        pricefeedVolatility: pricefeedVolatility,
```

Comment:
If this is a `BN` then you need to convert it to a string for the logging. Smack a `.toString()` on these two bad boys.

Reply:
done